Library/AddOn.cs

```diff
@@ -6,11 +6,16 @@ namespace Recurly
 {
     public class AddOn : RecurlyEntity
     {
-
         public string PlanCode { get; set; }
         public string AddOnCode { get; set; }
         public string Name { get; set; }
-
+        public int DefaultQuantity { get; set; }
+        public bool? DisplayQuantityOnHostedPage { get; set; }
+        public string TaxCode { get; set; }
+        public bool? Optional { get; set; }
+        public String AccountingCode { get; set; }
```

Comment:
Should we add `xmlWriter.WriteElementString("accounting_code", AccountingCode);` [further down](https://github.com/recurly/recurly-client-net/pull/154/files#diff-e84c4a9c2623e09a15d1c2b70fecc921R164)?